### PR TITLE
Bill Xin, set up Privacy Policy, change button to footer#34

### LINKED ISSUE #34: 2nd pr: Privacy Policy add html to page
# User Story

* As a user
* I can see a privacy policy
* So that I know how my data will be used

# Discussion

It is important for all systems that process sensitive information to disclose how that information will be used.

# Acceptance Criteria

- [ ] There is a new Page component called <PrivacyPage>
- [ ] This page component has placeholder text:  `<h2>Privacy Policy</h2>`
- [ ] Beneath the `h2` tag, the custom HTML below should be included to publish a privacy policy from termly.io
- [ ] This page is available at the link `/privacy` in the application (this is done in `App.js`)
- [ ] The `<Footer>` component has the text `Privacy Policy` somewhere suitable, with a link to `/privacy`

# Custom HTML

Custom HTML is in the file: 

https://github.com/ucsb-cs156/proj-gauchoride/blob/main/frontend/public/privacy.html

This may be copied directly into the PrivacyPage component.

### PR DESCRIPTION
**Pull request by abcdefucsb**
As a user
I can see a privacy policy
So that I know how my data will be used

closes https://github.com/ucsb-cs156-m23/proj-gauchoride-m23-9am-2/issues/34

image

image

### CODE REVIEW

**Review by andrewthaii — DISMISSED** [review]
Looks good. Approving.

**Review by pconrad — CHANGES_REQUESTED** [review]
This acceptance criterion from Issue #34  has not been met:

- [ ]  The <Footer> component has the text Privacy Policy somewhere suitable, with a link to /privacy

Please remove the "Privacy Policy" button from the Navbar (where it was never supposed to be in the first place), and instead, add a link in the Footer, which is where the customer wants it to be.

**Comment by abcdefucsb** [comment]
"privacy policy" is moved to the footer. My initial comment for this pr has been updated

File: frontend/src/main/pages/PrivacyPolicy.js [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+
 export default function PrivacyPolicy() {
   return (
     <BasicLayout>
       <div className="pt-2">
         <h2>Privacy Policy</h2> 
         <p>
-            PLACEHOLDER HTML FILE WILL BE INCLUDED
+            
+            <a href="privacy.html">Go to Privacy Policy</a>
```
Comment (pconrad) [inline frontend/src/main/pages/PrivacyPolicy.js:11]: Put the contents of `privacy.html`, which you deleted, here.

**Review by pconrad — CHANGES_REQUESTED** [review]
Why was privacy.html deleted?  

What I would have expected was that you would put that content into the PrivacyPolicy.js file.

Make sure that when you run the application, and click on the privacy policy link in the footer, that what comes up is actually the privacy policy.

**Comment by abcdefucsb** [comment]
Modified the project with Dr. Conrad. My initial comment has been updated.

**Review by Mohamed-Elfouly — APPROVED** [review]
LGTM

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**